tasks/settings.yml

```diff
@@ -52,4 +75,5 @@
   service: name=jenkins state=restarted
   when: (jenkins_users_config is defined and jenkins_users_config.changed) or
         (jenkins_http_config is defined and jenkins_http_config.changed) or
+        (jenkins_process_user_config is defined and jenkins_process_user_config.changed) or
```

Comment:
Except that the change suggested doesn't work because the setting requires a single value, and the suggested change produces a space separated list of values